app/Http/Controllers/TemperatureController.php

```diff
@@ -15,4 +16,13 @@ public function latest()
     {
         return Temperature::getLastRecord();
     }
+
+    public function store(Request $request)
```

Comment:
I tried to handle model creation in more elegant way, please remake the others in the same way.